docs/userGuide/syntax/textStyles.md

```diff
@@ -26,12 +26,40 @@ Syntax added by MarkBind:
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">markdown</variable>
 <variable name="code">
-****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~
+****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--,
+Super^script^, Sub~script~,
 ->Center-align<-
 </variable>
 </include>
 </div>
 
+Additional font colouring syntax:
+<div id="main-example-markbind">
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+#r#Coloured Text##
+</variable>
+</include>
+
+Full list of colours:
+Letter | Colour
+--- | ---
+r | red
+g | green
+b | blue
+c | cyan
+m | magenta
+y | yellow
+k | black
+w | white
+
+<box type="tip" seamless>
+To escape the syntax, simply put a backslash in front of it (e.g. `\#b#`, `\##`).
```

Comment:
Likely need to leave a blank line as the markdown is not rendered properly
![image](https://user-images.githubusercontent.com/41845017/192739504-098bbb1d-1395-4d51-a0d3-b1e6b8677b5a.png)